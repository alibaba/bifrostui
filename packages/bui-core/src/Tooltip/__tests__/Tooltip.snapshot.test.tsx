import { snapshotTest } from 'testing';

describe('Tooltip demo snapshot', () => {
  snapshotTest('Tooltip');
});
