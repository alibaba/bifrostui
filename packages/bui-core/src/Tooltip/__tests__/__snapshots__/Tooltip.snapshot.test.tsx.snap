// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Tooltip demo snapshot Tooltip demo snapshot 0 1`] = `
<span
  onClick={[Function]}
>
  click显示
</span>
`;

exports[`Tooltip demo snapshot Tooltip demo snapshot 0 2`] = `
<span
  onClick={[Function]}
>
  onOpenChange 回调方法
</span>
`;
